raster-foundry/raster-foundry
# Remove sttp from notification

## Overview

This PR removes sttp from the notification package. It was causing some problems in the batch container with missing methods, so it's been replaced with the http4s client, which is confirmed more cooperative in previous upgrades, for example in the integration test for scopes.

### Checklist


## Notes

I processed both successfully, so I expect that this will _actually work_ 🤞🏻

## Testing Instructions

- assemble api, batch
- bring up servers
- bring up groundwork pointing to local if you don't have a handy upload or stac export to try
- upload an image
- process the upload: `./scripts/process-upload <the id>`
- create a STAC export for the campaign
- process the export: `./scripts/console batch "java -cp /opt/raster-foundry/jars/batch-assembly.jar com.rasterfoundry.batch.Main write_stac_catalog <the id for your export>"`
- both async tasks should succeed, notify you where you expect, and not blow up with missing circe method errors
